README.md

```diff
@@ -268,10 +268,14 @@ Try [seeding the database](https://github.com/DFE-Digital/teaching-vacancies#see
 
 ### Getting production-like data for local development
 
-You can use conduit to create a dump of production data. See [this section](/documentation/hosting.md#backuprestore-govuk-paas-postgres-service-database) of the GovUK PaaS docs. Then you can load this into your local database:
+To get sanitised production-like data for local development, first log in to AWS with the ReadOnly role. To do so, follow the instructions here: [AWS Login](/documentation/aws-roles-and-cli-tools.md#log-in-to-the-aws-console-with-aws-vault).
+
+Once logged in, go to S3 >  530003481352-tv-db-backups > sanitised. Then click the checkbox next to the backup you want (the names of the backups will include dates) and click "Download".
```

Comment:
what is this 5300... number - is it your user ref number? if it is should prob be more generic. if not ignore this!

Reply:
It's just the name of the folder. I think it's given a number because it needs to be globally unique or something?